src/stacks-hierarchy/Root_FareContractDetailsScreen.tsx

```diff
@@ -26,6 +27,13 @@ export function Root_FareContractDetailsScreen({navigation, route}: Props) {
   const firstTravelRight = fc?.travelRights[0];
   const {t} = useTranslation();
 
+  useEffect(() => {
+    RCTPassPresentation.requestAutomaticPassPresentationSuppression();
+    return () => {
+      RCTPassPresentation.endAutomaticPassPresentationSuppression();
+    };
+  }, []);
```

Comment:
I think this could be defined as a custom hook and placed inside the `suppress-pass-presentation` module.

Reply:
Good catch 👍